zebra-chain/src/orchard_zsa/burn.rs

```diff
@@ -166,7 +164,15 @@ impl From<Burn> for ValueCommitment {
         burn.0
             .into_iter()
             .map(|BurnItem(asset, amount)| {
-                ValueCommitment::with_asset(pallas::Scalar::zero(), amount, &asset)
+                ValueCommitment::with_asset(
+                    pallas::Scalar::zero(),
+                    // FIXME: consider to use TryFrom and return an error instead of using "expect"
```

Comment:
This should be a u64 so the conversion should be infallible.

Reply:
That makes sense. But to make it work, the new `ValueCommitment::with_asset` constructor I added earlier should accept the value as a `NoteValue` instead of `Amount` (which is i64-based). I've fixed that and pushed the change into the `zsa-integration-state` branch.